doc/performance-investigations/0023-upgrade-python-to-3.11.4/summary.md

```diff
@@ -0,0 +1,33 @@
+# Python 3.11.4 Upgrade investigation
+
+EQ Questionaire Runner is currently on `3.10.8` .
+This investigation is to see the performance of Runner when upgraded to `3.11.4` compared to `3.10.8` .
+
+https://github.com/ONSdigital/eq-questionnaire-runner/tree/upgrade-python-version-311
+
+## Benchmark profile
+
+| Option | Value |
+|--------|-------|
+| Requests file | requests/test_benchmark_business.json|
+| Run time | 20m |
+| User wait time minimum | 1s |
+| User wait time maximum | 2s |
+| Clients | 50 |
+| Hatch rate | 50 |
+
+## Results
+
+Results based on 99th percentile timings.
+| Metric              | Baseline | Investigation |
```

Comment:
Any significant change in CPU/Mem between the two?.

Reply:
Pretty much the same:
<img width="471" alt="Screenshot 2023-09-05 at 09 15 27" src="https://github.com/ONSdigital/eq-survey-runner-benchmark/assets/53475968/e27d70b5-1525-4d8f-95c2-46beda47e8e0">
<img width="452" alt="Screenshot 2023-09-05 at 09 15 37" src="https://github.com/ONSdigital/eq-survey-runner-benchmark/assets/53475968/c0496777-095d-4aa5-a0e8-b4d4da931a51">